packages/cell-toolbar-extension/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@jupyterlab/cell-toolbar-extension",
+  "version": "4.0.0-alpha.6",
```

Comment:
FYI `4.0.0a22` has been released, so this (and other references) can be re-aligned to:

```suggestion
  "version": "4.0.0-alpha.7",
```

Reply:
However it shouldn't be needed to publish again to npm.